<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelPortaAtual">
      <property name="text">
       <string>Porta atual:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDiretorioAtual">
      <property name="text">
       <string>Diretorio atual:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBandaAtual">
      <property name="text">
       <string>Banda atual:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Porta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="linePorta"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Diretorio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDiretorio"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Velocidade (B/s) - Digite 'n' para nao limitar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineBanda"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushAplicar">
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushEncerrar">
      <property name="text">
       <string>Encerrar Servidor</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
